extensions/websockets-next/server/deployment/src/test/java/io/quarkus/websockets/next/test/subprotocol/SubprotocolSelectedTest.java

```diff
@@ -27,7 +27,7 @@ public class SubprotocolSelectedTest {
     public static final QuarkusUnitTest test = new QuarkusUnitTest()
             .withApplicationRoot(root -> {
                 root.addClasses(Endpoint.class, WSClient.class);
-            }).overrideConfigKey("quarkus.websockets-next.supported-subprotocols", "oak,larch");
+            }).overrideConfigKey("quarkus.websockets.next.server.supported-subprotocols", "oak,larch");
```

Comment:
why did we switched to `.next` ?

Reply:
Hm, I thought that the convention is use dots in similar cases but you're right that it does not make a lot of sense in this particular case so I'll revert this change but keep the additional `.server` part which is IMO practical.

I.e. `quarkus.websockets-next.supported-subprotocols` will become `quarkus.websockets-next.server.supported-subprotocols`.

Reply:
Perfect, yes, I was referring to the dot between websockets and next. For `.server`, it  makes sense. 

We should never forget YAML.... (while I hope to forget all about it every single day)